src/ApiPort/ConsoleProgressReporter.cs

```diff
@@ -32,6 +38,28 @@ public void ReportIssue(string issue)
             _issuesReported.Add(issue);
         }
 
+        /// <summary>
+        /// Suspends all progress tasks.
+        /// </summary>
+        public void SuspendProgressTasks()
```

Comment:
Suspend/Resume might be better names. Is there value in suffixing with ProgressTasks?